src/Components/Header.js

```diff
@@ -9,6 +10,10 @@ const Header = () => (
       <FaAngleLeft className="Icon" />
     </NavLink>
     <h3><a href="https://www.indmoney.com/us-stocks/top-tech-stocks">US Technology Company</a></h3>
+    <div>
+      <FiSettings />
+      <FiMic />
+    </div>
```

Comment:
- [ ] Let's follow the project guidelines, the project design should match the original, and pay attention to typography, space between elements, color, etc.

![layout](https://user-images.githubusercontent.com/53124548/217781972-4c1fb0bb-2cee-49f7-a1d2-9577b11a61ac.png)

![home-finance](https://user-images.githubusercontent.com/53124548/217782085-cc1589c9-a09d-411a-a2e4-7aa362c67a83.png)

Reply:
Well resolved ✔️